android/java/org/uproxy/tun2socks/Tunnel.java

```diff
@@ -126,6 +126,8 @@ public synchronized void stop() {
 
   private static final String VPN_INTERFACE_NETMASK = "255.255.255.0";
   private static final int VPN_INTERFACE_MTU = 1500;
+  private static final String DNS_RESOLVER_IP = "8.8.8.8";
```

Comment:
I'd rather add issues for these. 
* Multiple DNS servers is more of a feature. Is the second for fallback?
* A change like this is likely to include some UI component if the user is to choose these servers.